runtime/src/lib.rs

```diff
@@ -159,6 +192,21 @@ pub const EPOCH_DURATION_IN_BLOCKS: BlockNumber = 1 * HOURS;
 pub const UNIT: Balance = 1_000_000_000_000;
```

Comment:
do we want to use these units in 'master' branch instead of `CENTS`, etc, for consistency?

Reply:
I have removed this. Its not needed